README.md

```diff
@@ -18,6 +18,63 @@ yarn dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
+Artsy team members should copy over the .env values from 1pass to get the app to fully work. For our open source friends, some parts of the app may not function.
+
+## Providing a page layout
+
+A concept introduced by this app which isn't normally apart of Next.js is the ability to provide a layout for all pages in a directory.
+Layouts for next.js pages are powered by files named `_layout.tsx`. For example, all the pages in `pages/team` share the same layout thanks to `pages/team/_layout.tsx`. When building, every page will look for a layout file in its directory or in any parent directory up to the pages directory. It will use the nearest `_layout` file to determine its layout (if one is present).
+
+You can manually override the layout of any given page by attaching `.Layout` to the component. For example:
+
+```jsx
+const Page = () => {
+  return <p>Test Page</p>;
+};
+
+// This will override any _layout file
+Page.Layout = ({ children }) => (
+  <>
+    <h1>This is a test</h1>
+    {children}
+  </>
+);
+
+export default Page;
+```
+
+If you just want to _disable_ the layout for that file you can just set layout equal to null like `Page.Layout = null`.
+
+## Customizing mdx pages
+
+Layouts for mdx files are stored in the `/layouts` directory.
```

Comment:
q: Can `mdx` pages live inside of `/pages/`? If yes, why do mdx layouts have to live in `/layouts`? Thats the one part of this that I think we should strive to unify, in effect removing the `/layouts` folder completely so that there are no longer two diverging concepts at play.

Reply:
That's... complicated. The short answer is right now, no. There's two particular reasons for this. 

1. We're using [next-mdx-enhanced](https://github.com/hashicorp/next-mdx-enhanced#readme) which prescribes this layout mechanism. Given how the plugin current works, we're not going to be able to get it to work in `pages` until we make some under-the-hood changes. I'm not opposed to doing those, it'll just take some time. That leads me to...
2. As mentioned in the PR body, the zeit team is pretty strongly against any file being in `pages` that's not directly a page. This is primarily for security purposes as I understand it. I've only gotten around that with `_layout` by hackily automatically generating a default export that's actually a 404 page, heh. I _can_ do the same thing for mdx files (ish) it's just going to take a bit of planning, forking, and hacking to get there.

Reply:
I see. If it's just not possible right now then np 👍 The two layout concepts feel like friction but its not that big of a deal.